src/plugins/action_update_review_status/src/updateReviews.ts

```diff
@@ -0,0 +1,72 @@
+import databaseMethods from './databaseMethods'
+
+type ReviewStatus = 'Draft' | 'Submitted' | 'Changes Requested' | 'Pending' | 'Locked'
+interface Review {
+  reviewId: number
+  reviewAssignmentId: number
+  applicationId: number
+  reviewerId: number
+  level: number
+  reviewStatus: ReviewStatus
+}
+
+module.exports['updateReviews'] = async function (input: any, DBConnect: any) {
+  const db = databaseMethods(DBConnect)
+
+  console.log('Updating reviews status...')
+
+  const { applicationId, applicationData, changedApplicationResponses = [] } = input
+  const stageId = input?.stageId || applicationData.stageId
+  const level = input?.level || applicationData?.reviewData?.level || 1
```

Comment:
hmm, is there a way to do additions in evaluator (as the level to check is + 1).

Still requires a little tweak (might have been better done in another issue, so we can merge this one. Maybe we shouldn't rely at all on the `input`, since it's hard to name the level input properly (since it's + 1 of the related review).

something like:
```
const reviewLevel = applicationData?.reviewData?.level || 0
const level = reviewLevel + 1
```

Reply:
Yeah, we can ignore the input, sure, I'll just remove it from this expression.

I'm slightly confused about the other bit -- if an application is submitted, we want to update all the level 1 reviews, correct? And if a level 1 review is submitted, we also want to update the other level 1 reviews, don't we? And if a level 2 review is submitted, we want to update the other L2 reviews? Or is it update the current level reviews AND the one above?

Reply:
>  And if a level 1 review is submitted, we also want to update the other level 1 reviews, don't we?
 
 In this action, we just want to update lvl + 1 (or lvl 1 review) <- this action should really be about notifying reviewers that something below their level has happened that needs their attention.

I would imagine there would be another action that would set status of reviews to locked ?

Reply:
I am going to approve this for now, and we can change it when we get to consolidation level

Reply:
> I am going to approve this for now, and we can change it when we get to consolidation level

Yeah cool, I think it will be easier to make small tweaks to it once we're working with real applications in the UI. 

Cheers.